### PR TITLE
Update zbxTelegram.py

### PR DESCRIPTION
**Pull request by ozaryx**
Add set token from command args. Token from args has high priority. This will allow to configure telegram bot token directly from zabbix.

### CODE REVIEW

File: zbxTelegram.py [hunk 1]
```diff
@@ -63,7 +63,12 @@ def __missing__(self, key):
 args = ArgParsing().create_parser().parse_args(sys.argv[1:])
 loggings = System(config_debug_mode if not args.debug else True).log
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+if args.token:
+    tg_token = args.token
+
 bot = telebot.TeleBot(tg_token)
```
Comment (xxsokolov) [inline zbxTelegram.py:70]: bot = telebot.TeleBot(args.token if args.token else tg_token)

**Review by xxsokolov — CHANGES_REQUESTED** [review]
change

File: zbxTelegram.py (hunk 1 above)
Comment (ozaryx) [reply zbxTelegram.py:70]: Ок. It's look better
bot = telebot.TeleBot(args.token if args.token else tg_token)

**Comment by ozaryx** [comment]
Make another PR

**Pull request closed without merging**